hosts

```diff
@@ -1,6 +1,8 @@
 [caliopen:vars]
-use_https=false
+use_https=true
+stage=true
 vhost_name=poc.caliopen.org
+compose_file="{{ '/opt/caliopen/devtools/docker-compose.staging.yml' if stage == 'true' else '/opt/caliopen/devtools/docker-compose.yml' }}"
```

Comment:
Maybe there is a better way to make this variable available for both the main playbook and the load data one.